tests/rptest/services/redpanda.py

```diff
@@ -505,12 +505,16 @@ def __init__(self,
                  security: SecurityConfig = SecurityConfig(),
                  node_ready_timeout_s=None,
                  superuser: Optional[SaslCredentials] = None,
-                 skip_if_no_redpanda_log: bool = False):
+                 skip_if_no_redpanda_log: bool = False,
+                 pp_keep_alive: Optional[int] = None,
+                 pp_cache_max_size: Optional[int] = None):
         super(RedpandaService, self).__init__(context, num_nodes=num_brokers)
         self._context = context
         self._extra_rp_conf = extra_rp_conf or dict()
         self._enable_pp = enable_pp
         self._enable_sr = enable_sr
+        self._pp_keep_alive = pp_keep_alive
+        self._pp_cache_max_size = pp_cache_max_size
```

Comment:
I think it's time that we create a python wrapper for all the PP/SR configs. I can do that in the next push or delay to another PR.

Reply:
https://github.com/redpanda-data/redpanda/issues/6878